medusa/storage/ibm_storage.py

```diff
@@ -59,62 +54,7 @@ def connect_storage(self):
         if self.config.transfer_max_bandwidth is not None:
             self.set_upload_bandwidth()
 
-        return driver
-
-    def get_object_datetime(self, blob):
-        logging.debug(
-            "Blob {} last modification time is {}".format(
-                blob.name, blob.extra["last_modified"]
-            )
-        )
-        return parser.parse(blob.extra["last_modified"])
-
-    def get_cache_path(self, path):
-        # Full path for files that will be taken from previous backups
-        return path
-
-    def check_dependencies(self):
-        if self.config.aws_cli_path == 'dynamic':
-            aws_cli_path = AwsCli.find_aws_cli()
-        else:
-            aws_cli_path = self.config.aws_cli_path
-
-        try:
-            subprocess.check_call([aws_cli_path, "help"], stdout=PIPE, stderr=PIPE)
-        except Exception:
-            raise RuntimeError(
-                "AWS cli doesn't seem to be installed on this system and is a "
-                + "required dependency for the S3 backend. Please install it by running 'pip install awscli' "
-                + "or 'sudo apt-get install awscli' and try again."
-            )
+        if self.config.host is None and self.config.region is not None:
+            self.config.host = IBM_CLOUD_HOSTS_BY_REGION[self.config.region]
```

Comment:
You cannot change a config element like this. But I think we should drop this static mapping and let the user fill in the host and region in config.